docs/quick-visual-tutorials/visual-ibac-istio-authorization-policies.mdx

```diff
@@ -8,15 +8,325 @@ import CodeBlock from "@theme/CodeBlock";
+<img src="/img/quick-tutorials/istio-visual-tutorial/frontend-productactalogservice-missing-intent.png" alt="Discovered intents" width="600"/>
+
+- This access is not blocked *now* &mdash; because we're still in shadow mode (otherwise the line would have been red).
+- But access *would be blocked* once enforcement is turned on. To prevent that, we're told to declare and apply an intent for this call.
+
+Click on the `productcatalogservice`:
+<img src="/img/quick-tutorials/istio-visual-tutorial/productcatalog-would-block.png" alt="Discovered intents" width="600"/>
+
+- We can see it's not protected now, as before.
+- But we can also see it *would* block any clients once protection is enabled, which is why the lock is red.
+- And there is an explicit warning to apply the missing intents from all its clients before turning on enforcement.
+
+
+Let's add those intents for the `frontend` and `checkoutservice`.
+
+<Tabs>
+<TabItem value="frontend" label="frontend" default>
+
+```yaml
+{@include: ../../static/code-examples/ibac-for-istio/phase-3-frontend.yaml}
+```
+</TabItem>
+<TabItem value="checkout-service" label="checkoutservice" default>
+
+```yaml
+{@include: ../../static/code-examples/ibac-for-istio/phase-3-checkout.yaml}
+```
+</TabItem>
+</Tabs>
+
+Apply these intents files with:
+```bash
+kubectl apply -n otterize-visual-tutorial-istio -f https://docs.otterize.com/code-examples/ibac-for-istio/phase-3.yaml
+```
+
+Let's go back to the access graph:
+
+![Access graph](/img/quick-tutorials/istio-visual-tutorial/phase-3.png)
+
+Click on the `productcatalogservice`:
+
+![Access graph](/img/quick-tutorials/istio-visual-tutorial/productcatalog-service-all-green.png)
+
+All is well again: the `productcatalogservice` will be protected, and its 3 clients will have access after enforcement is turned on.
+Each client access is limited to the HTTP resource and method declared in the intent.
+
+:::tip We can see how to roll out IBAC gradually:
+1. Pick a service to protect.
+2. Make sure all its clients declare and intents to call it.
+3. When you're ready, turn on enforcement.
+
+The access graph and shadow mode allow us to gain confidence by showing what would happen and highlighting any problems.
+:::
+
+
+### Protect everything easily
+Could we somehow automatically bootstrap this for the whole cluster and protect all services, without breaking any intended calls? Yes!
+
+The network mapper keeps track of all attempted calls, after all: those are the discovered intents. If you are confident that all of those calls are intended and appropriate, you can use that information to automatically generate intent declarations and apply them.
+
+Let's use the Otterize CLI ([installation](/installation#install-the-otterize-cli) and [reference](/reference/cli) to export all discovered intents as YAML declarations:
+```bash
+otterize network-mapper export -n otterize-visual-tutorial-istio --output-type dir --output intents
+```
+
```

Comment:
Would be cool to show the exported intents, to make it explicit as to what they look like, and to emphasize the work that was saved.

Reply:
I know we have them in the docs repo, and we reference them below, but it would be cool to actually show them, either as a whole bunch of tabs, or in a single long YAML with scrolling.